examples/caption_generation.ipynb

```diff
@@ -0,0 +1,1277 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    fd.run(overwrite=True)
</pre><p>Perhaps make a note that overwrite=True is an optional argument and remove it from fd.run? Users don't need the argument to run the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/visual-layer/fastdup/pull/272/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
thanks, added a comment